conf/cassandra.yaml

```diff
@@ -1302,6 +1302,11 @@ slow_query_log_timeout: 500ms
 #    Snitch for getting dc and rack of a node from metadata service of Alibaba cloud.
 #    This snitch that assumes an ECS region is a DC and an ECS availability_zone is a rack.
 #
+# AzureSnitch:
```

Comment:
nit: 
```
# AzureSnitch:
#    Gets datacenter from 'location' and rack from 'zone' fields of 'compute' object
#    from instance metadata service. If the availability zone is not enabled, it will use the fault
#    domain and get its respective value.
#
```